rust/src/bin/coordinator.rs

```diff
@@ -9,7 +9,7 @@ use tracing_futures::Instrument;
 
 #[cfg(feature = "influx_metrics")]
 use xain_fl::{
-    common::metric_store::influxdb::{run_metricstore, InfluxDBMetricStore},
+    common::metric_store::influxdb::{run_metricstore, InfluxDBConnector},
```

Comment:
Why the renaming? I'm don't mind it I'm just curious about the reason.

Reply:
I thought having the name metric store twice in the path is a bit too much. 
My plan was actually to do something like this:
```
common::metric_store::{
	run_metricstore,
	influxdb::Connector
}
```
But I wasn't sure if this change makes so much sense if we only have one metric store implementation.

Reply:
I think it makes sense. I don't know if we'll have other implementations but it doesn't hurt to plan for it.

Reply:
I have created a issue for that #347